### DISCUSSION TITLE
What is Difference Policies and Rules?

### DISCUSSION THREAD

**Discussion by mehdihadeli** [kgrzybek/modular-monolith-with-ddd#182] [Q&A] [answered]
Hi @kgrzybek,
What is Difference Policies and Rules in your code base?

**Comment by kgrzybek** [comment, marked as answer]
Hi @mehdihadeli,

- Rule: a business rule which describes **business invariant** (something, which should be always true)
- Policy: an object which represent other than the business invariant rule and does not belong to entity/vo. Most of the time it will be some processing rule (from EventStorming view - **if something happened (event) then do something (command)**)

**Reply by mehdihadeli** [reply]
@kgrzybek Thanks.
